corehq/apps/app_manager/models.py

```diff
@@ -1442,6 +1442,8 @@ def add_property_load(self, app_case_meta, case_type, name,
                 "%s is not a valid question" % question_path
             )
 
+    @quickcache(['self.unique_id', 'self.version'], timeout=24 * 60 * 60,
```

Comment:
Is it complicated to bust cache? (say, on an app-change in the domain)

Reply:
an app change will automatically bump the `version`, so it wouldn't hit the cache anyways